pkg/controller/controller.go

```diff
@@ -560,37 +560,58 @@ func (c *Controller) syncHandler(key string) error {
 			}
 		}
 
-		// Get apim instance 1 service name using hardcoded value
-		apimingressname := "wso2-am-p1-ingress"
-		amingress, err := c.ingressLister.Ingresses(apimanager.Namespace).Get(apimingressname)
-		// If the resource doesn't exist, we'll create it
-		if errors.IsNotFound(err) {
-			amingress, err = c.kubeclientset.ExtensionsV1beta1().Ingresses(apimanager.Namespace).Create(pattern1.ApimIngress(apimanager))
-			if err != nil {
-				return err
+		if apimanager.Spec.Expose == "Ingress" {
+			// Get apim instance 1 service name using hardcoded value
+			apimingressname := "wso2-am-p1-ingress"
```

Comment:
shall we make var names camel case?

Reply:
Fixed with https://github.com/wso2/k8s-wso2am-operator/pull/77